src/SearchWrapper.cpp

```diff
@@ -10,358 +11,360 @@
 using namespace std;
 
 Rcpp::List
-convertToDataFrame(Rcpp::List list)
-{
+  convertToDataFrame(Rcpp::List list)
```

Comment:
Try not to commit whitespace changes in the same commit as other changes (particularly if the whitespace changes aren't actually wanted; these don't feel right to me compared to the previous code you've written).